ldclient/client.py

```diff
@@ -35,8 +35,8 @@ class Config(object):
     def __init__(self,
                  base_uri='https://app.launchdarkly.com',
                  events_uri='https://events.launchdarkly.com',
-                 connect_timeout=2,
-                 read_timeout=10,
+                 connect_timeout=10,
```

Comment:
These were lower than what we've seen in the field (5 seconds just for dns lookup) so I'm upping them here.